src/components/Header/index.jsx

```diff
@@ -8,7 +8,7 @@ import {
   Work as WorkIcon,
 } from '@mui/icons-material';
 import { AppBar, Button, IconButton, Tab, Tabs, Toolbar, Typography } from '@mui/material';
-import GordonDialogBox from 'components/GordonDialogBox';
+import GordonDialogBox from 'components/GordonDialogBox/index';
```

Comment:
is there a specific reason this was changed?

Reply:
I don't remember why this was changed either but I can switch it back